vllm-tensorizer/Dockerfile

```
@@ -53,12 +68,20 @@ WORKDIR /workspace
 
 RUN apt-get -qq update && apt-get install -y --no-install-recommends curl && apt-get clean
 
-RUN pip3 install "transformers == 4.36.0" "fschat[model_worker] == 0.2.30" "triton == 2.1.0"
+RUN --mount=type=bind,from=freezer,target=/tmp/frozen \
+    /tmp/frozen/freeze.sh torch torchaudio torchvision xformers > /tmp/constraints.txt
+
+RUN python3 -m pip install \
+      "fschat[model_worker] == 0.2.30" "triton == 2.1.0" \
+      -c /tmp/constraints.txt
```

This one is missing `--no-cache-dir`
```suggestion
RUN python3 -m pip install --no-cache-dir \
      "fschat[model_worker] == 0.2.30" "triton == 2.1.0" \
      -c /tmp/constraints.txt
```